lms/templates/navigation.html

```diff
@@ -78,6 +79,22 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
     </ol>
     <ol class="user">
       <li class="primary">
+          <form action="/i18n/setlang/" method="post" class="settings-language-form" id="settings-form">
+            <input type="hidden" id="csrf_token" name="csrfmiddlewaretoken" value="${csrf_token}">
+            <input title="preference api" aria-label="language" type="hidden" id="preference-api-url" value="${reverse('preferences_api', kwargs={'username': user.username})}">
+            <label for="settings-language-value"><span class="sr">${_("Choose Language")}</span></label>
```

Comment:
I'd prefer to keep the ```id``` attribute on this select element because it's helpful with DOM work as well.